spec/cc/workspace/path_tree_spec.rb

```diff
@@ -58,7 +69,7 @@ class CC::Workspace
       within_temp_dir do
         make_fixture_tree
 
-        tree = PathTree.new(".")
+        tree = PathTree.for_path(".")
         tree.exclude_paths(["code/does-not-exist"])
         expect(tree.all_paths.sort).to eq [".git/", "code/a/", "code/foo.rb", "foo.txt", "lib/"]
```

Comment:
If you're suggesting that attempting to exclude a non-existent path should blow up, we can't really do that. The use case is that you may have files locally you want excluded, but those files may be in your `.gitignore`, so they won't exist on .com.